src/Umbraco.Web.UI.Client/src/common/directives/components/umbdatetimepicker.directive.js

```diff
@@ -0,0 +1,143 @@
+/**
+@ngdoc directive
+@name umbraco.directives.directive:umbDateTimePicker
+@restrict E
+@scope
+
+@description
+Use this directive to render a date time picker
+
+<h3>Markup example</h3>
+<pre>
+	<div ng-controller="My.Controller as vm">
+
+        <umb-date-time-picker
+            options="vm.config"
+            ng-model="vm.date"
+            on-change="vm.datePickerChange(event)"
+            on-error="vm.datePickerError(event)">
+        </umb-date-time-picker>
+
+	</div>
+</pre>
+
+<h3>Controller example</h3>
+<pre>
+	(function () {
+		"use strict";
+
+		function Controller() {
+
+            var vm = this;
+
+            vm.date = "";
+
+            vm.config = {
+                pickDate: true,
+                pickTime: true,
+                useSeconds: true,
+                format: "YYYY-MM-DD HH:mm:ss",
```

Comment:
I don't think we should just modify the dates inside the directive? I can see it uses moments for something but haven't really spend time on it.

The docs doesn't show all the config options but I have added a link to the original documentation.